nextcord/application_command.py

```diff
@@ -837,16 +886,13 @@ async def call_invoke_slash(
 
         await self.invoke_slash(interaction, **kwargs)
 
-    # This method is only meant to be called by nextcord.Client
-    async def _send_error(self, interaction: Interaction, error: ApplicationError) -> None:
+    async def dispatch_error(self, interaction: Interaction, error: ApplicationError) -> None:
 
         if self.has_error_handler():
```

Comment:
Hmm, arguable if the dispatch/invoker should be checking if it has an error handler instead of just throwing an exception. Up to you on that though.

Reply:
It's done this way because the `application_error` event receives the error alongside the application command error handler. Raising an exception inside this function would break the error handling system sorta